### ISSUE TITLE
Manual omissions + lack of basic use examples

### ISSUE THREAD

**Issue by neokio** [cartalyst/stripe#83]
A few thoughts:

1) There's no mention of the mandatory namespace declaration in the "[Instantiation](https://cartalyst.com/manual/stripe/2.0#instantiation)" section of the manual (or anywhere). Without calling `namespace Cartalyst\Stripe;` before `$stripe = new Stripe();`, nothing will work.

2) Also in the "[Instantiation](https://cartalyst.com/manual/stripe/2.0#instantiation)" section, `$stripe = new Stripe('your-stripe-api-key', 'your-stripe-api-version');` is shown as the preferred method. There's no mention of acceptable values, or whether or not it's required. In every other code example, api-version is omitted, which is confusing.

3) In the "[Customers](https://cartalyst.com/manual/stripe/2.0#customers)" section, `source` is listed as a required parameter.  Yet the usage example doesn't use it. Is it required or not?

Basically, the manual lacks examples of basic, real-world use. No start-to-finish "Hello world" code. I'd suggest creating a few super-simple example .php files to really help help people hit the gate running. Adding a few files to /vendor/cartalyst/stripe/examples/ would do wonders.

**Comment by brunogaspar** [comment]
Hi @neokio 

You're more than welcomed to submit PR's to improve the documentation.

**Comment by neokio** [comment]
I probably will, once I actually know what I'm talking about :)
But at the moment I'm not qualified. I'm just getting started with the Stripe API today.

**Comment by brunogaspar** [comment]
Sure.

I'll give you some feedback on the points on mentioned.

--

> There's no mention of the mandatory namespace declaration in the "Instantiation" section of the manual (or anywhere). Without calling namespace Cartalyst\Stripe; before $stripe = new Stripe();, nothing will work.

Agree. 

I would probably have the `use Cartalyst\Stripe\Stripe;` on each usage example.

> Also in the "Instantiation" section, $stripe = new Stripe('your-stripe-api-key', 'your-stripe-api-version'); is shown as the preferred method. There's no mention of acceptable values, or whether or not it's required. In every other code example, api-version is omitted, which is confusing.

Agree.

This can probably be done a tad differently to avoid confusion.

> In the "Customers" section, source is listed as a required parameter. Yet the usage example doesn't use it. Is it required or not?

Most likely a typo, it should be `false`.

> Basically, the manual lacks examples of basic, real-world use. No start-to-finish "Hello world" code. I'd suggest creating a few super-simple example .php files to really help help people hit the gate running. Adding a few files to /vendor/cartalyst/stripe/examples/ would do wonders.

Not sure i agree with this one though, the documentation has examples for every single thing it can do, so it's sufficient.

--- 

I'll try to tackle some of these during the weekend though.

**Comment by neokio** [comment]
Another ... in the "[Charges](https://cartalyst.com/manual/stripe/2.0#charges)" section, it lists `source` as required, but the usage example doesn't include a `source`.

**Comment by brunogaspar** [comment]
Thanks.

I've fixed the incorrect requirement values and added the namespace imports on the instantiation.

Didn't worked on the key and version part though as i've started to re-organise the documentation sections.

Going to close this though, but feel free to leave comments if you find any other occurrences or even do a PR if you can of course.
